src/transformers/models/layoutlmv3/modeling_layoutlmv3.py

```diff
@@ -886,7 +886,9 @@ def forward(
                 position_ids = position_ids.expand_as(input_ids)
                 final_position_ids = position_ids
 
-        extended_attention_mask: torch.Tensor = self.get_extended_attention_mask(attention_mask, None, device)
+        extended_attention_mask: torch.Tensor = self.get_extended_attention_mask(
+            attention_mask, None, device, dtype=embedding_output.dtype
+        )
```

Comment:
Need to pass the `embedding_output.dtype` in order to make `LayoutLMv3ModelTest.test_multi_gpu_data_parallel_forward` works.